integration-tests/profiler.spec.js

```diff
@@ -65,6 +72,97 @@ describe('profiler', () => {
     await sandbox.remove()
   })
 
+  it('code hotspots and endpoint tracing works', async () => {
+    const procStart = BigInt(Date.now() * 1000000)
+    const proc = fork(path.join(cwd, 'profiler/codehotspots.js'), {
+      cwd,
+      env: {
+        DD_PROFILING_PROFILERS: 'wall',
+        DD_PROFILING_EXPORTERS: 'file',
+        DD_PROFILING_ENABLED: 1,
+        DD_PROFILING_CODEHOTSPOTS_ENABLED: 1,
+        DD_PROFILING_ENDPOINT_COLLECTION_ENABLED: 1
+      }
+    })
+
+    await processExitPromise(proc, 5000)
+    const procEnd = BigInt(Date.now() * 1000000)
```

Comment:
How do you ensure that the right `wall_.+.pprof` is selected if they are several ones ? 
Maybe select the latest one ?
This logic could probably be extracted into a function.

Reply:
yes, sandbox is cleared between execution of the whole `describe('profiler', ...` suite, but if you change the order of the tests inside this suite, it might break...